frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -30,6 +30,25 @@ export const createLookup = <A extends K8sResourceKind>(
   return {};
 };
 
+export const getRandomUUID = (): string => {
```

Comment:
Lets add a comment that this implementation is a direct replacement for crypto.randomUUID(), and it will generate a cryptographically secure, UUID v4-compliant identifier, for environments where crypto.randomUUID() may not be available.